tests/fsharpqa/Source/Conformance/DeclarationElements/AccessibilityAnnotations/basic/InterfaceImplementationVisibility.fs

```diff
@@ -0,0 +1,84 @@
+
+module Definitions =
```

Comment:
That could be an additional test, but for the purpose of this fix I was trying to make things as clear as possible by keeping everything public except the interfaces/types themselves.